QUEST_LV_0100.tsv

```diff
@@ -7960,7 +7960,7 @@ QUEST_LV_0100_20150908_007955	Putting the oil
 QUEST_LV_0100_20150908_007956	The magic supplier has been fixed{nl}Activate the secondary purifier
 QUEST_LV_0100_20150908_007957	The secondary purifier is working again
 QUEST_LV_0100_20150908_007958	It seems that a part of the purifier is missing{nl}Use the mine compass to look for the part
-QUEST_LV_0100_20150908_007959	The mine compass is pointing at 6th districte{nl}Look for the parts at 6th district
+QUEST_LV_0100_20150908_007959	The mine compass is pointing at 6th District{nl}Look for the parts at 6th District
```

Comment:
I'm willing to do it, but I'm first checking the upcoming areas. Though, I think the Crystal Mine looks pretty good so far.